app/src/main/java/org/simple/clinic/newentry/PatientEntryScreenController.kt

```diff
@@ -92,7 +93,7 @@ class PatientEntryScreenController @Inject constructor(
                   district = facility.district,
                   state = facility.state))
         }
-        .map { { ui: Ui -> ui.preFillFields(it) } }
+        .flatMap { Observable.never<UiChange>() }
```

Comment:
Instead of doing this, why don't we mark this method as `@Deprecated` and just remove this from being part of the merged observable chain in the `apply()` method. That way, all the code paths in this will never get executed as well.

Reply:
Discussed offline: May not be valid in all cases since these might be mutating state in between.